_posts/en/newsletters/2018-09-25-newsletter.md

```diff
@@ -0,0 +1,204 @@
+---
+title: 'Bitcoin Optech Newsletter #14'
+permalink: /en/newsletters/2018/09/25/
+name: 2018-09-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes action items and news related to last week's
+security release of Bitcoin Core 0.16.3 and Bitcoin Core 0.17RC4, popular
+questions and answers from the Bitcoin StackExchange over the past
+month, and short descriptions of notable merges made to popular Bitcoin
+infrastructure projects.
+
+- **Upgrade to Bitcoin Core 0.16.3 to fix CVE-2018-17144:** as widely
+  reported early Friday (UTC), the denial-of-service vulnerability
+  described in last week's Optech newsletter is now known to allow
+  miners to trick affected systems into accepting invalid bitcoins.
+  
+    As of this writing, it's believed that a majority of large Bitcoin
```

Comment:
What is the source of this belief?

Reply:
For businesses, I'm the source based on my belief that they've had enough time to upgrade, could suffer massive losses, and are basically sane.  For miners, the source is the BitcoinCore.org notice linked in this post that says they believe a majority of miners have upgraded.

Reply:
I've personally emailed several of the large exchanges and had confirmation that they've upgraded.